erpnext/stock/doctype/item/item.json

```diff
@@ -3937,6 +3937,38 @@
    "set_only_once": 0, 
    "translatable": 0, 
    "unique": 0
+  }, 
+  {
+   "allow_bulk_edit": 0, 
+   "allow_in_quick_entry": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "amazon_item_code", 
```

Comment:
just revert these files to the original, and we can merge!

Reply:
done